browser/resources/settings/default_brave_shields_page/brave_adblock_browser_proxy.ts

```diff
@@ -23,92 +43,75 @@ export enum ErrorCode {
-  addWebUiListener(eventName: string, callback: Function)
-  updateSubscription(url: string)
-  deleteSubscription(url: string)
-  viewSubscription(url: string)
-  getCustomScriptlets(): Promise<Scriptlet[]>
-  addCustomScriptlet(scriptlet: Scriptlet): Promise<ErrorCode>
-  updateCustomScriptlet(name: string, scriptlet: Scriptlet): Promise<ErrorCode>
-  removeCustomScriptlet(name: string): Promise<ErrorCode>
+  getRegionalLists: () => Promise<FilterList[]>
+  enableFilterList: (uuid: string, enabled: boolean) => void
+  updateFilterLists: () => Promise<boolean>
+  getCustomFilters: () => Promise<string>
+  updateCustomFilters: (value: string) => void
+  getListSubscriptions: () => Promise<any>
+  setSubscriptionEnabled: (url: string, enabled: boolean) => void
+  addSubscription: (url: string) => void
+  updateSubscription: (url: string) => void
+  deleteSubscription: (url: string) => void
+  viewSubscription: (url: string) => void
+  getCustomScriptlets: () => Promise<Scriptlet[]>
+  addCustomScriptlet: (scriptlet: Scriptlet) => Promise<ErrorCode>
+  updateCustomScriptlet: (name: string, scriptlet: Scriptlet) =>
+    Promise<ErrorCode>
+  removeCustomScriptlet: (name: string) => Promise<ErrorCode>
+  addWebUiListener: (eventName: string, callback: Function) => void
 }
 
 export class BraveAdblockBrowserProxyImpl implements BraveAdblockBrowserProxy {
   static getInstance() {
-    return instance || (instance = new BraveAdblockBrowserProxyImpl());
+    return instance || (instance = new BraveAdblockBrowserProxyImpl())
   }
 
-  /** @returns {Promise} */
-  getRegionalLists () {
+  getRegionalLists (): Promise<FilterList[]> {
     return sendWithPromise('brave_adblock.getRegionalLists')
   }
 
-  /** @returns {Promise} */
-  getListSubscriptions () {
-    return sendWithPromise('brave_adblock.getListSubscriptions')
+  enableFilterList (uuid: string, enabled: boolean): void {
+    chrome.send('brave_adblock.enableFilterList', [uuid, enabled])
+  }
+
+  updateFilterLists (): Promise<boolean> {
+    return sendWithPromise('brave_adblock.updateFilterLists')
   }
 
-  /** @returns {Promise} */
-  getCustomFilters () {
+  getCustomFilters (): Promise<string> {
     return sendWithPromise('brave_adblock.getCustomFilters')
   }
 
-  enableFilterList (uuid, enabled) {
-    chrome.send('brave_adblock.enableFilterList', [uuid, enabled])
+  updateCustomFilters (value: string): void {
+    chrome.send('brave_adblock.updateCustomFilters', [value])
   }
 
-  /** @returns {Promise<boolean>} */
-  updateFilterLists () {
-    return sendWithPromise('brave_adblock.updateFilterLists')
+  getListSubscriptions (): Promise<any> {
+    return sendWithPromise('brave_adblock.getListSubscriptions')
   }
 
-  setSubscriptionEnabled (url, enabled) {
+  setSubscriptionEnabled (url: string, enabled: boolean): void {
     chrome.send('brave_adblock.setSubscriptionEnabled', [url, enabled])
   }
 
-  addSubscription (url) {
+  addSubscription (url: string): void {
```

Comment:
note: void is implied, so its generally best to omit it (here and in other places)


```suggestion
  addSubscription (url: string) {
```

Reply:
Fixed here https://github.com/brave/brave-core/pull/27107/commits/392900e38135b0f651fa26f4e889b04e04d66471